vignettes/compactify.Rmd

```diff
@@ -102,7 +102,7 @@ speeds <- rbind(speeds, speed_test(iterate_as_of,"as_of_1000x"))
 
 # Performance of slide
 slide_median <- function(my_ea) {
-    my_ea$slide(median = median(case_rate_7d_av), n = 7)
+    my_ea$slide(median = median(case_rate_7d_av), before = 7)
```

Comment:
Unlike the above, where, after changing `before_num` to be `= before`, we would want to adjust the `before` arguments to be 120->119, 3->2, and 2->1, we probably want to keep before=7 here, because for jhu-csse cases, we typically expect time values up to v-1 in each version v, so before=7 would have 7 time values and be averaging across a week.  (Of course, it doesn't make too much sense to be 7-day-averaging a 7-day-average, but that might be a tweak for a separate issue.)